src/main/java/com/edmunds/tools/databricks/maven/model/ClusterTemplateModel.java

```diff
@@ -0,0 +1,105 @@
+/*
+ *  Copyright 2019 Edmunds.com, Inc.
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License");
+ *      you may not use this file except in compliance with the License.
+ *      You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *      Unless required by applicable law or agreed to in writing, software
+ *      distributed under the License is distributed on an "AS IS" BASIS,
+ *      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *      See the License for the specific language governing permissions and
+ *      limitations under the License.
+ */
+
+package com.edmunds.tools.databricks.maven.model;
+
+import com.edmunds.rest.databricks.DTO.AwsAttributesDTO;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Collection;
+
+/**
+ * Simple POJO to pass properties to the cluster template.
+ */
+public class ClusterTemplateModel {
```

Comment:
Should this be in the plugin, or the rest-client?

Reply:
Suddenly found such class in rest-client - ClusterAttributesDTO. Will use it. Thanks.

Reply:
good call out